addons/website/controllers/main.py

```diff
@@ -274,7 +274,7 @@ def get_dynamic_filter(self, filter_id, template_key, limit=None, search_domain=
         dynamic_filter = request.env['website.snippet.filter'].sudo().search(
             [('id', '=', filter_id)] + request.website.website_domain()
         )
-        return dynamic_filter and dynamic_filter.render(template_key, limit, search_domain, with_sample) or ''
+        return dynamic_filter and dynamic_filter.render(template_key, limit, search_domain, with_sample, request.params) or ''
```

Comment:
only used in action server, so you can use request.params directly and avoid go give this param from function to function